### PR TITLE
NET Framework metrics

### PR DESCRIPTION
**Pull request by thiagoftsm**
This PR is bringing `.NET ` available metrics.

<details> <summary>Images from metrics that are not zero</summary>

</details>

### CODE REVIEW

**Comment by thiagoftsm** [comment]
This PR was set as draft again, because we will reorganize menus before to merge it.

File: modules/wmi/charts.go [hunk 1]
```diff
@@ -2175,6 +2234,512 @@ var (
...
+		netFrameworkCLRRemotingCalls.Copy(),
+		netFrameworkCLRSecurityLinkTimeChecks.Copy(),
+		netFrameworkCLRSecurityChecksTime.Copy(),
+		netFrameworkCLRSecurityStackWalkDepth.Copy(),
+		netFrameworkCLRSecurityRuntimeChecks.Copy(),
+	}
+
+	// Exceptions
+	netFrameworkCLRExceptionsThrown = module.Chart{
+		OverModule: "NET Framework",
```
Comment (ilyam8) [inline modules/wmi/charts.go:2288]:
```suggestion
		OverModule: "netframework",
```

File: modules/wmi/collect_netframework.go [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+	metricNetFrameworkCLRSecurityRTChecksTimePercent = "windows_netframework_clrsecurity_rt_checks_time_percent"
+	metricNetFrameworkCLRSecurityStackWalkDepth      = "windows_netframework_clrsecurity_stack_walk_depth"
+	metricNetFrameworkCLRSecurityRuntimeChecksTotal  = "windows_netframework_clrsecurity_runtime_checks_total"
+)
+
+func (w *WMI) collectNetFrameworkCLR(mx map[string]int64, pms prometheus.Series) {
+	if !w.cache.collection[collectorNetFrameworkCLR] {
+		w.cache.collection[collectorNetFrameworkCLR] = true
+		w.addNetFrameworkCRLExceptions()
+	}
```
Comment (ilyam8) [inline modules/wmi/collect_netframework.go:65]: I see that `netframework_*` are [different collectors](https://github.com/prometheus-community/windows_exporter#collectors). You need to create specific collectors charts only if it is enabled (as it is now - you create all `netframework_*` charts if `netframework_clrexceptions` is enabled).

File: modules/wmi/charts.go [hunk 3]
```diff
@@ -2175,6 +2234,512 @@ var (
...
+	netFrameworkCLRExceptionsThrown = module.Chart{
+		OverModule: "netframework",
+		ID:         "net_framework_exception_thrown",
+		Title:      "Number of exceptions thrown since start.",
+		Units:      "exceptions",
+		Fam:        "exceptions",
+		Ctx:        "net_framework.exception_thrown",
+		Type:       module.Stacked,
+		Priority:   prioNETFrameworkCLRExceptionsThrown,
+	}
```
Comment (ilyam8) [inline modules/wmi/charts.go:2296]: I think we discussed it already. Please do either summary charts (+= every process value) or a chart per process.

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: modules/wmi/collect_netframework.go (hunk 2 above)
Comment (thiagoftsm) [reply modules/wmi/collect_netframework.go:65]: I will split all them. :handshake:

File: modules/wmi/charts.go (hunk 3 above)
Comment (thiagoftsm) [reply modules/wmi/charts.go]: Done, PR is ready for review.

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**